src/fidesctl/core/config/logging_settings.py

```diff
@@ -0,0 +1,52 @@
+"""This module handles finding and parsing fides configuration files."""
+
+# pylint: disable=C0115,C0116, E0213
+import os
+from logging import DEBUG, INFO, getLevelName
+from typing import Union
+
+from pydantic import validator
+
+from .fides_settings import FidesSettings
```

Comment:
```suggestion
from fideslib.core.config import FidesSetttings
```

Reply:
I thought about making this change but our base classes have different configs and I don't think we really gain anything by sharing this base class atm. Our base config class currently forbids extra configs but the fideslib config is set to allow. 

@ThomasLaPiana @sanders41 thoughts?

```
    class Config:
        # Need to allow extras because the inheriting class will have more info
        extra = Extra.allow

```

Reply:
Right fideslib has to allow extras since the other libraries will extend it. The nice thing about the shared base is making sure things stay in sync, but as you say the base currently doesn't do much of anything so it isn't a big deal right now. I would say this isn't a blocker and lets do whatever makes the most sense now. We can always re-visit it if things change in the future.